<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>516</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="TitleRecorridoConductor">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>50</y>
     <width>411</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
     <weight>75</weight>
     <italic>true</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>RECORRIDOS POR CONDUCTOR</string>
   </property>
  </widget>
  <widget class="QLabel" name="lfechaFinalRC">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>170</y>
     <width>131</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
     <weight>75</weight>
     <italic>false</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>FECHA FINAL</string>
   </property>
  </widget>
  <widget class="QLabel" name="lfechaInicialRC">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>130</y>
     <width>141</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
     <weight>75</weight>
     <italic>false</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string> FECHA INICIAL</string>
   </property>
  </widget>
  <widget class="QDateEdit" name="fechaInicialRC">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>130</y>
     <width>151</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="lIDConductorRC">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>210</y>
     <width>151</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
     <weight>75</weight>
     <italic>false</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>ID CONDUCTOR</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineIDConductorRC">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>210</y>
     <width>151</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="buscarRH">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>260</y>
     <width>401</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>CONSULTAR</string>
   </property>
  </widget>
  <widget class="QDateEdit" name="fechaInicialRC_2">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>170</y>
     <width>151</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
